src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.String.cs

```diff
@@ -41,6 +46,39 @@ public static partial class JsonSerializer
                 throw new ArgumentNullException(nameof(json));
             }
 
+            return Deserialize<TValue>(json.AsSpan(), typeof(TValue), options);
+        }
+
+        /// <summary>
+        /// Parse the text representing a single JSON value into a <typeparamref name="TValue"/>.
+        /// </summary>
+        /// <returns>A <typeparamref name="TValue"/> representation of the JSON value.</returns>
+        /// <param name="json">JSON text to parse.</param>
+        /// <param name="options">Options to control the behavior during parsing.</param>
+        /// <exception cref="JsonException">
+        /// The JSON is invalid.
+        ///
+        /// -or-
+        ///
+        /// <typeparamref name="TValue" /> is not compatible with the JSON.
+        ///
+        /// -or-
+        ///
+        /// There is remaining data in the span beyond a single JSON value.</exception>
+        /// <exception cref="NotSupportedException">
+        /// There is no compatible <see cref="System.Text.Json.Serialization.JsonConverter"/>
+        /// for <typeparamref name="TValue"/> or its serializable members.
+        /// </exception>
+        /// <remarks>Using a UTF-16 span is not as efficient as using the
+        /// UTF-8 methods since the implementation natively uses UTF-8.
+        /// </remarks>
+        public static TValue? Deserialize<[DynamicallyAccessedMembers(MembersAccessedOnRead)] TValue>(ReadOnlySpan<char> json, JsonSerializerOptions? options = null)
+        {
+            if (json == null)
+            {
+                throw new ArgumentNullException(nameof(json));
```

Comment:
[This Comment](https://github.com/dotnet/runtime/pull/41957#discussion_r487592893) suggests a special test for default span. Otherwise, default span will be treated same as empty span.
What's the final decision on this?

Reply:
> Otherwise, default span will be treated same as empty span.

Default and empty spans **should** be treated the same. Adding a test passing in default span is good to have, but the null check within the source implementation isn't necessary and should be removed.

In the implementation of Deserialize, a default/empty span should be allowed to be passed to the `Utf8JsonReader`, and it will validate that empty JSON is invalid and throw `JsonException`. That is the desired behavior.